### PR TITLE
structure for dumpResults function

### PR DESCRIPTION
**Pull request by pidoshva**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -19,27 +19,28 @@ int main() {
...
+    cout << "30XX: Add a word from a specific location in memory to the word in the accumulator and leave the result in the accumulator"<<endl;
+    cout << "31XX: Subtract a word from a specific location in memory from the word in the accumulator and leave the result in the accumulator"<<endl;
+    cout << "32XX: Divide the word in the accumulator by a word from a specific location in memory and leave the result in the accumulator"<<endl;
+    cout << "33XX: Multiply a word from a specific location in memory to the word in the accumulator and leave the result in the accumulator"<<endl;
+    cout << "40XX: Branch to a specific location in memory."<<endl;
+    cout << "41XX: Branch to a specific location in memory if the accumulator is negative"<<endl;
+    cout << "42XX: Branch to a specific location in memory if the accumulator is zero"<<endl;
+    cout << "4300: Halt command"<<endl;
+    cout << "Input -99999 to stop inputing commands"<<endl;
+    cout<<"^---!!!PLEASE SEE THE REQUIREMENTS FOR WELCOME MESSAGE!!!---^"<<endl;
```
Comment (jansenfuller) [inline main.cpp:43]: Do we really need a message saying to read the instructions?

File: memory.cpp [hunk 2]
```diff
@@ -16,7 +16,7 @@ int Memory::get_value(int mem_location) {
 }
 
 void Memory::print_memory() {
-    cout<<"MEMORY"<<endl; cout<<"\t00\t01\t02\t03\t04\t05\t06\t07\t08\t09"<<endl; // Column count
+    cout<<"\nMEMORY"<<endl; cout<<"\t00\t01\t02\t03\t04\t05\t06\t07\t08\t09"<<endl; // Column count
```
Comment (jansenfuller) [inline memory.cpp:19]: What's with all of these "\t00\t01" characters?

Comment (pidoshva) [reply memory.cpp:19]: https://dev.azure.com/CS-2450-Summer2022/UVSim/_backlogs/backlog/UVSim%20Team/Features/?workitem=6


> What's with all of these "\t00\t01" characters?

File: main.cpp (hunk 1 above)
Comment (pidoshva) [reply main.cpp:43]: > Do we really need a message saying to read the instructions?

https://dev.azure.com/CS-2450-Summer2022/UVSim/_backlogs/backlog/UVSim%20Team/Features/?workitem=2

File: memory.cpp (hunk 2 above)
Comment (pidoshva) [reply memory.cpp:19]: I changed the number of tabs for everything to line up in memory visualization table.

Comment (jansenfuller) [reply memory.cpp:19]: Gotcha, sets the tabs

**Pull request closed without merging**